### DISCUSSION TITLE
STATUS CODE ERROR 403 on B&H

### DISCUSSION THREAD

**Discussion by spencerhochhaus** [jef/streetmerchant#2117] [Help] [answered]
How can I set a custom page_sleep_min and page_sleep_max for just one store or product page? All the other stores give me green info but B&H quickly gives me yellow warnings as said in the title. Thank you in advance.

**Comment by breacher18** [comment, marked as answer]
E.g.: amazon:10000:30000,bestbuy,newegg:50000:80000.

**Reply by spencerhochhaus** [reply]
Thank you so much

**Reply by spencerhochhaus** [reply]
Do you have any ideas on the best reload time for B&H is? I keep getting the error code despite >30s reload min times.
